python/tests/test_account.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import print_function
+from deltachat.capi import lib
+
+
+class TestOfflineAccount:
+    def test_selfcontact(self, acfactory):
+        ac1 = acfactory.get_offline_account()
+        me = ac1.get_self_contact()
+        assert me.display_name
+        # assert me.addr  # xxx why is this empty?
```

Comment:
looking in the source code, this happens if the context is not yet configured.
configure() changes and normalizes the config-options; this may also be true for the email address (spaces, maybe umlaut encoding (although not yet done) and such stuff).